Server/RFIDNamespace.py

```diff
@@ -17,11 +22,39 @@ class RFID(Resource):
     @api.response(200, "success")
     @api.response(404, "Unknown Card")
     def get(self, card_id):
-        # TODO: Actually check a database for the known cards.
-        if card_id != "8666529cc":
+        user = User.query.filter_by(card_id = card_id).first()
+        if not user:
             return UNKNOWN_CARD_RESPONSE
         else:
             return "Welcome back!"
 
-
+# name, email are required for new users. Ability can be passed multiple times.
+# Only adds abilities; does not remove them.
+@RFID_namespace.route("/update/<string:card_id>/")
+@RFID_namespace.doc(description ="Update a user's information")
+class RFID(Resource):
```

Comment:
MYPY typing is complaining that RFID is defined twice.  

You can see this when running ./checks.sh

Reply:
Oops! I wonder why the tests didn't whine about that... fixed now